requests/dev version of urllib3 don't play nice together in tox

Working with a local clone of current master (ae4adde):

```console
$ tox -e docs
GLOB sdist-make: /Users/alexwlchan/repos/urllib3/setup.py
docs create: /Users/alexwlchan/repos/urllib3/.tox/docs
docs installdeps: -r/Users/alexwlchan/repos/urllib3/docs/requirements.txt
docs inst: /Users/alexwlchan/repos/urllib3/.tox/dist/urllib3-dev.zip
docs installed: alabaster==0.7.10,appdirs==1.4.3,asn1crypto==0.22.0,Babel==2.4.0,backports.ssl-match-hostname==3.5.0.1,certifi==2017.4.17,cffi==1.10.0,chardet==3.0.3,coverage==3.7.1,cryptography==1.8.2,docutils==0.13.1,enum34==1.1.6,funcsigs==1.0.2,idna==2.5,imagesize==0.7.1,ipaddress==1.0.18,Jinja2==2.9.6,MarkupSafe==1.0,mock==1.3.0,ndg-httpsclient==0.4.2,nose==1.3.7,nose-exclude==0.4.1,packaging==16.8,pbr==3.0.1,pkginfo==1.4.1,pluggy==0.3.1,psutil==4.3.1,py==1.4.33,pycparser==2.17,Pygments==2.2.0,pyOpenSSL==17.0.0,pyparsing==2.2.0,PySocks==1.5.6,pytz==2017.2,requests==2.16.5,six==1.10.0,snowballstemmer==1.2.1,Sphinx==1.6.2,sphinxcontrib-websupport==1.0.1,tornado==4.2.1,tox==2.1.1,twine==1.5.0,typing==3.6.1,urllib3===dev,virtualenv==15.1.0
docs runtests: PYTHONHASHSEED='3695751977'
docs runtests: commands[0] | rm -rf /Users/alexwlchan/repos/urllib3/docs/_build
docs runtests: commands[1] | make -C /Users/alexwlchan/repos/urllib3/docs html
sphinx-build -b html -d _build/doctrees  '-W' . _build/html
Running Sphinx v1.6.2
making output directory...

Exception occurred:
  File "/Users/alexwlchan/repos/urllib3/.tox/docs/lib/python2.7/site-packages/requests/__init__.py", line 53, in <module>
    major, minor, patch = urllib3_version
ValueError: need more than 1 value to unpack
The full traceback has been saved in /var/folders/jy/351n9lnj5l3f07rtf2xybxx00000gn/T/sphinx-err-dn6eJg.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 1
ERROR: InvocationError: '/usr/bin/make -C /Users/alexwlchan/repos/urllib3/docs html'
________________________________________________________________________________ summary _________________________________________________________________________________
ERROR:   docs: commands failed
```

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -2,3 +2,4 @@
 ndg-httpsclient
 sphinx
 alabaster
+requests>=2.0,<2.16

